### PR TITLE
Example cases

### PR DESCRIPTION
**Pull request by kylemas**
- New pull request for changes in the handleSubmit method under PremisesSelector component. 
- Changed FormEventHandler to ChangedEventHandler
- Changed HTMLFormElement to HTMLSelectElement
- Reverted type of handleSubmitPremises on App.tsx back to 'string' from 'any'. 
- Changed definition of premises variable under handleSubmit on PremisesSelector.

### CODE REVIEW

File: src/components/PremisesSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { ChangeEventHandler, FormEventHandler } from 'react'
+
+type Props = {
+    onChange: (premises: string) => void
+}
+
+const PremisesSelector: React.FC<Props> = ({ onChange }) => {
+    const handleSubmit: ChangeEventHandler<HTMLSelectElement> = (event) => {
+        event.preventDefault()
+        const premises: string = (event.currentTarget.value)
```
Comment (McTano) [inline src/components/PremisesSelector.tsx]: Do you have prettier set up to format automatically?
It should remove these brackets, for example.

Comment (McTano) [reply src/components/PremisesSelector.tsx]: The formatting in this file is off, actually. It's using four space indentation instead of 2.

Comment (McTano) [reply src/components/PremisesSelector.tsx]: I'm going to configure our repo to lint automatically when we commit.

Comment (kylemas) [reply src/components/PremisesSelector.tsx]: I didn't notice that it was disabled. I've enabled it now & fixed the formatting issues.

Comment (kylemas) [reply src/components/PremisesSelector.tsx]: Sorry about that.

**Comment by McTano** [comment]
There are still eslint errors on the latest commit.
Run `yarn eslint` or use vscode's problems view (the little (X) and /!\ icons on the bottom panel to see the eslint errors.

**Comment by kylemas** [comment]
> There are still eslint errors on the latest commit.
> Run `yarn eslint` or use vscode's problems view (the little (X) and /!\ icons on the bottom panel to see the eslint errors.

Screen Shot 2019-08-14 at 4 04 08 PM
Screen Shot 2019-08-14 at 4 04 39 PM

I tried running 'yarn eslint' as well as checking the problems view on vscode. I'm not really sure what I'm looking for?

**Comment by McTano** [comment]
Sorry, that should be yarn lint.

**Comment by McTano** [comment]
When you open the PremisesSelector file in vscode, you should be able to see the errors highlighted.
Do you have the eslint extension installed?

**Comment by McTano** [comment]
Since the PremisesSelector file is open in your screenshot, you should be able to see the errors in problems view. It must not be set up right.

**Comment by kylemas** [comment]
> Since the PremisesSelector file is open in your screenshot, you should be able to see the errors in problems view. It must not be set up right.

Yeah, I didn't have the eslint extension installed. I fixed up the issues/warnings now!

File: src/components/PremisesSelector.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const PremisesSelector: React.FC<Props> = ({ onChange }) => {
     <select onChange={handleSubmit}>
       <option value="P->Q,P,~Q">Basic Example</option>
       <option value="P->Q">Modus Ponens</option>
-      <option value="~PvQ">De Morgen's Law</option>
+      <option value="~PvQ">De Morgens Law</option>
```
Comment (McTano) [inline src/components/PremisesSelector.tsx]: You do want it to say "De Morgen's", though.  Not "De Morgens".

As the eslint error says,
```
`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.eslint(react/no-unescaped-entities)
```

Comment (McTano) [reply src/components/PremisesSelector.tsx]: use `&apos;` in place of the apostrophe.

**Pull request merged**